src/gltf-component-mappings.js

```diff
@@ -470,7 +470,26 @@ AFRAME.GLTFModelPlus.registerComponent(
       }
     }
 
-    el.setAttribute(componentName, { ...componentData, srcEl });
+    // Migrate audio-target component that still has builting audio params
+    if (componentData.positional !== undefined) {
```

Comment:
Hmm, if the audio type is not positional then we won't actually add an `audio-params` object, but I think other code in `audio-source` like `createAudio()` assumes one always exists...

Reply:
The blender exported added that parameter for `audio-target` (before the `audio-params` refactor) so we consider that if that exists this is an old `audio-target` component and we create the `audio-params` component.

https://github.com/MozillaReality/hubs-blender-exporter/pull/29/commits/1be80cccfc0101017f9fd033a30c1a564cd17ae1#diff-a07a63e062368f64b62670e35510bc7194e1c23c69aaa0732d25586ff734eee6R795-R798

Not sure why we used that instead of the audioType enum that we used in other components.

Anyways I'm' closing this PR in  favor of https://github.com/mozilla/hubs/pull/4444  as we restoring the stereo support there and was missing here.